bugsnag-plugin-android-exitinfo/src/test/java/com/bugsnag/android/TombstoneEnhancerTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.bugsnag.android
+
+import android.app.ApplicationExitInfo
+import com.bugsnag.android.internal.ImmutableConfig
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when`
+import org.mockito.junit.MockitoJUnitRunner
+import com.bugsnag.android.Thread as BugsnagThread
+
+@RunWith(MockitoJUnitRunner::class)
```

Comment:
This test doesn't appear to make sure of any Mockito annotations (`@Mock` and friends) which makes this `@RunWith` redundant (so it can be removed).